### PR TITLE
Streams 1.6.3

### PR DESCRIPTION
**Pull request by maronov-amzn**
*Description of changes:*
- CCP configuration + audio settings (https://code.amazon.com/packages/AmazonConnectStreams/commits/33909d3baae9d479bae57d69825489a9593d0b7e)

### CODE REVIEW

File: Documentation.md [hunk 1]
```diff
@@ -664,7 +712,7 @@ Subscribe a method to be invoked when the contact is pending. This event is expe
 ```js
 contact.onConnecting(function(contact) { /* ... */ });
 ```
-Subscribe a method to be invoked when the contact is connecting. This event happens when a contact comes in, before accepting (there is an exception for queue callbacks, in which onConnecting's handler is started after the callback is accepted). Note that once the contact has been accepted, the `onAccepted` handler will be triggered.
+Subscribe a method to be invoked when the contact is connecting. This event happens when a contact comes in, before accepting (there is an exception for queue callbacks, in which onConnecting's handler is executed after the callback is accepted). Note that once the contact has been accepted, the `onAccepted` handler will be triggered.
```
Comment (martin-greenberg) [inline Documentation.md:715]: non-inclusive language was put back here

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "amazon-connect-streams",
-  "version": "1.6.2",
+  "name": "@amzn/amazon-connect-streams",
```
Comment (martin-greenberg) [inline package.json:2]: need to remove @amzn prefix

File: src/index.d.ts [hunk 3]
```diff
@@ -761,11 +785,61 @@ declare namespace connect {
...
+
+    /**
+     * Sets the ringer device (output device for ringtone)
+     *
+     * @param deviceId The id of the media device.
+     */
+    setRingerDevice(deviceId: string): void;
+
+    /**
+     * Subscribe a method to be called when the agent updates the mute status, meaning that agents mute/unmute APIs are called and the local media stream is succesfully updated with the new status.
```
Comment (martin-greenberg) [inline src/index.d.ts:809]: typo was restored here "successfully" -> "succesfully"

**Review by martin-greenberg — CHANGES_REQUESTED** [review]

File: Documentation.md [hunk 4]
```diff
@@ -626,14 +637,51 @@ agent.unmute();
...
+
 ### `agent.onMuteToggle()`
 ```js
 agent.onMuteToggle(function(obj) {
    if (obj.muted) { /* ... */ }
 });
 ```
 Subscribe a method to be called when the agent updates the mute status, meaning
-that agents mute/unmute APIs are called and the local media stream is successfully updated with the new status.
+that agents mute/unmute APIs are called and the local media stream is succesfully updated with the new status.
```
Comment (ratsirarar) [inline Documentation.md:666]: typo `succesfully`

**Review by ratsirarar — CHANGES_REQUESTED** [review]

**Review by martin-greenberg — APPROVED** [review]
LGTM

**Review by ratsirarar — APPROVED** [review]
LGTM

**Pull request merged**